doc/ref_model/chapters/chapter10.md

```diff
@@ -51,4 +52,3 @@ Over the past few years there has been a significant move towards decomposing ne
 
 ## 10.3.4 Packet Acceleration Request (e.g Hardware Acceleration) [Initial language]
```

Comment:
what is this sentence trying to say, in particular the bolded part, "The acceleration hardware is typically optimized for specific kinds of workloads and **some form of disaggregation might be required to separate control and user plane responsibilities between _generic server hardware and accelerators_**."

Reply:
@pgoyal01 - Please see my comment above. I agree with you. Even I think this sentence is bit confusing. However, do you want me to address it as part of this PR or should there be a seperate PR for this? 
Note I did not write this section.

Reply:
@sukhdevkapur Issue #1861 opened.